pkg/pidfile/pidfile.go

```diff
@@ -0,0 +1,51 @@
+package pidfile
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+)
+
+type PidFile struct {
```

Comment:
Yeah that does make sense. I wonder though whether nothing will be added in the future... Perhaps @crosbymichael has an idea about this?